cypress/integration/login.js

```diff
@@ -7,21 +7,61 @@ describe('Sourcebooks login', function() {
         cy.get('[type="submit"]').click();
         cy.get('.Select.not-valid').should('be.visible')
     })
+ 
+    const Roles =
+    [
+        {
+            name: "User",
+            tabs: '1'
+        },
+        {
+            name: "Team Lead",
+            tabs: '2'
+        },
+        {
+            name: "Manager",
+            tabs: '5'
+        },
+        {
+            name: "Accountant",
+            tabs: '5'
+        },
+        {
+            name: "Admin",
+            tabs: '6'
+        }
+    ];
+       
+    
+    Roles.forEach((role) => {
+        it(`Verify ${role.name} role can log in and should see appropriate tabs`, function() {
+            cy.visit('/');
+            cy.get('[id="loginForm.userId"]').click({force:true});
+            cy.get('[aria-label="Dainius Gaidamavičius"]').click();
+            cy.get('[id="loginForm.role"]').click({force:true});
+            cy.get(`[aria-label='${role.name}']`).click();
+            cy.get('[type="submit"]').click();
+            cy.get('.user-info__title').contains('Dainius Gaidamavicius'); 
+            cy.get('.main-nav').find('li').should('have.length', role.tabs);                      
+            cy.get('.main-nav__link--active').should('have.css','color', 'rgb(64, 76, 237)');
+        })
+    })  
 
-    it('Should be able to login with role User', function () {
-
-        cy.visit('/');
+   /* it('T-3: Admin creates new task', function () {

```

Comment:
You should not commit commented out code, if it's commented out probably that means that it should be deleted